build.gradle.kts

```diff
@@ -33,11 +33,16 @@ plugins {
 
     id("nebula.integtest") version "7.0.9" apply false
     id("me.champeau.gradle.jmh") version "0.5.2"
+    id("org.openrewrite.rewrite") version "3.1.0"
 }
 
 apply(plugin = "nebula.integtest-standalone")
 apply(plugin = "nebula.publish-verification")
 
+configure<org.openrewrite.gradle.RewriteExtension> {
```

Comment:
You should be able to get away with just `rewrite {` here. 
After you add the entry to the plugins block click this button in the IDE
![image](https://user-images.githubusercontent.com/8549033/113364312-02be9d00-9308-11eb-9761-2f56fc01d90c.png)

Then new extensions made available by the plugin are accessible.